.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules/
-test.cy.js/
\ No newline at end of file
+test.cy.js/
+/test-results/
+/playwright-report/
```

Comment:
Good to add playwright-report to .gitinore, but it was already added, so if you can just remove that folder completely